### PR TITLE
Product grid

### LINKED ISSUE #5: Product Grid
On the products page there should be a grid of products with their associated details.

### PR DESCRIPTION
**Pull request by jennazenk**
closes #5

### CODE REVIEW

**Comment by Octowl** [comment]
closes #5

**Comment by Octowl** [comment]
@mmlanders23 @coreygreen1108 we need a code review 💃

File: browser/js/products/product.factory.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+app.factory('Product', function ($http) {
```
Comment (coreygreenwald) [inline browser/js/products/product.factory.js]: Can you rename your factories "ProductFactory" so we can quickly identify it is a factory?

File: browser/js/products/product.states.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+app.config(function ($stateProvider) {
+    $stateProvider.state('products', {
+        url : '/products',
+        templateUrl : 'js/products/products.html',
+        controller : 'ProductListCrtl',
+        resolve: {
+            products: function(Product) {
```
Comment (coreygreenwald) [inline browser/js/products/product.states.js]: Like here.

**Pull request merged**